addon/components/insert-variable-card.js

```diff
@@ -7,49 +7,44 @@ import { v4 as uuidv4 } from 'uuid';
 import fetchCodeLists from '../utils/fetchData';
 
 export default class EditorPluginsInsertCodelistCardComponent extends Component {
-  @tracked variableTypes = [
-    'text',
-    'number',
-    'date',
-    'location',
-    'codelist'
-  ];
+  @tracked variableTypes = ['text', 'number', 'date', 'location', 'codelist'];
   @tracked selectedVariableType;
   @tracked showCard = true;
-  @tracked isCodelist = false
+  @tracked isCodelist = false;
   @tracked selectedCodelist;
 
   constructor() {
     super(...arguments);
     const config = getOwner(this).resolveRegistration('config:environment');
-    this.endpoint = config.insertCodelistPlugin.endpoint;
-    const { administrativeUnitUuid } = this.args.widgetArgs.options
+    this.endpoint = config.insertVariablePlugin.endpoint;
+    const { publisher } = this.args.widgetArgs.options;
     this.args.controller.onEvent('selectionChanged', this.selectionChanged);
-    this.fetchCodeList.perform(administrativeUnitUuid)
+    this.fetchCodeList.perform(publisher);
   }
 
   @action
   insert() {
     const uri = `http://data.lblod.info/mappings/${uuidv4()}`;
     const htmlToInsert = `
       <span resource="${uri}" typeof="ext:Mapping">
+        <span property="dct:source" content="${this.endpoint}"></span>
```

Comment:
probably makes more sense to treat the endpoint as a URI? so `resource="${this.endpoint}"`